Cms_frontend/src/components/Editor/EditorSidebar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const Container = styled.div`
+  background: #eeeeee;
+  width: 15%;
```

Comment:
i think viewport width most of the time are better because i think 15% takes 15% of the parent's width